### PR TITLE
modified readme

### PR DESCRIPTION
**Pull request by Goodylaptop**

### CODE REVIEW

**Comment by MASTERGROOT** [comment]
what

File: main.py [hunk 1]
```diff
@@ -2,7 +2,6 @@
 import test_module
 from unittest import TestProgram
 
-
```
Comment (MASTERGROOT) [inline main.py:5]: what is this

**Comment by MASTERGROOT** [comment]
@Goodylaptop

**Comment by Goodylaptop** [comment]
donno @MASTERGROOT

**Review by MASTERGROOT — APPROVED** [review]
never mind just a blank space. I approve anyway.

**Pull request merged**